### ISSUE TITLE
pdf.js v1.9.426 support

### ISSUE THREAD

**Issue by TZanke** [hypothesis/pdf.js-hypothes.is#14]
I am searching for a solution to annotate pdf files with pdf.js. Hypothes.is looks like a good solution, but this project does not support any newer pdf.js versions.

Should i move further to another project because this solution is abandoned, or is there any plan to reanimate this project?

**Comment by pkraker** [comment]
I'd be very interested in that as well.

**Comment by ajpeddakotla** [comment]
@TZanke Thanks for your interest. As you can see from Rob's comment above, the client can use newer versions of PDF.js, but there are certain anchoring issues that can occur you if you try and view annotations created in older versions of PDF.js. This is an issue we'll need to address before we can update the PDF.js version we ship with the browser extension. 

We'll keep you updated as we resolve that issue and update the version we ship with the browser extension.

**Comment by klemay** [comment]
Closing in favor of https://github.com/hypothesis/via/issues/81
